content/docs/rfcs/19/README.md

```diff
@@ -0,0 +1,51 @@
+---
+slug: 19
+title: 19/WAKU2-LIGHTPUSH
+name: Waku v2 Light Push
+status: raw
+editor: Oskar Thorén <oskar@status.im>
+contributors:
+---
+
+**Protocol identifier**: `/vac/waku/lightpush/2.0.0-alpha1`
+
+# Motivation and goals
+
+Light nodes with short connection windows and limited bandwidth wish to publish messages into the Waku network. Additionally, there sometimes is a need for confirmation that a message has been received "by the network".
+
+`19/LIGHTPUSH` is a request/reply protocol for this.
+
+# Payloads
+
+```protobuf
+message PushRequest {
+    WakuMessage message = 1;
+}
+
+message PushResponse {
+    boolean is_success = 1;
+    // Error messages, etc
+    string info = 2;
+}
+
+message PushRPC {
+    string request_id = 1;
+    PushRequest query = 2;
+    PushResponse response = 3;
+}
+```
+
+## Message relaying
+
+Nodes that respond to `PushRequests` MUST relay this via `RELAY` protocol. If they are unable to do so for some reason, they SHOULD return an error code in `PushResponse`.
```

Comment:
Will they always do so on the default pubsubTopic? Perhaps we can add an optional `pubsubTopic` to `PushRequest` in which case the responding node MUST relay the `WakuMessage` accordingly (default only if `pubsubTopic` unset).

Reply:
Ah yes, we want to specify pubsubTopic here

Reply:
added `pubsub_topic`. In relay spec we support multiple, but that seems like an unnecessary amplification factor to me at this stage